deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -99,7 +99,8 @@ export class NetworkStack extends Stack {
         // General interface endpoints
         const interfaceEndpoints = [
             InterfaceVpcEndpointAwsService.CLOUDWATCH_LOGS, // Push Logs from tasks
-            InterfaceVpcEndpointAwsService.CLOUDWATCH_MONITORING, // Pull Metrics from Migration Console 
+            InterfaceVpcEndpointAwsService.CLOUDWATCH_MONITORING, // Pull Metrics from Migration Console
+            InterfaceVpcEndpointAwsService.EBS_DIRECT, // Enable EBS Volumes
```

Comment:
Sorry for the back and forth, but can we remove this? Looks like it's only for snapshot operations https://docs.aws.amazon.com/ebs/latest/APIReference/API_Operations.html